maestro/chart/values.yaml

```diff
@@ -0,0 +1,130 @@
+# Default values for maestro.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: overture/maestro
+  tag: latest
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: NodePort
+  port: 11235
+
+ingress:
+  enabled: false
+  annotations: {}
+  hosts:
+    - host: maestro.local
+      paths: []
+
+  tls: []
+
+resources: {}
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+appConfig:
+  server:
+    port: 80
+  maestro:
+    kafka:
+      # REPLACE this with KAFKA host/IP
+      broker: 10.11.7.31:9092
+      indexing_topic: maestro_index_requests
```

Comment:
camelCase in yaml

Reply:
currently the application expects this format .. I can't change them here without changing the app

Reply:
application.yml and values.yaml are two different things, they have different concerns. The former is read by Java app, while, values.yaml is used to generate Kubernetes resources, it pass the values to deployment.yaml, then deployment.yaml care about the application.yml